Framework/DataObjects/inc/MantidDataObjects/WorkspaceCreation.h

```diff
@@ -163,7 +169,8 @@ std::unique_ptr<T> create(const P &parent, const IndexArg &indexArg,
     }
   }
 
-  // The instrument is also copied by initializeFromParent, but if indexArg is
+  // The instrument is also copied by initializeFromParent or
+  // initializeFromParentWithoutLogs, but if indexArg is
```

Comment:
This change to the comment is confusing since there is no call to `initializeFromParentWithoutLogs` in this function.